test/test_images/latencymako_aggregator/main.go

```diff
@@ -107,7 +108,7 @@ func main() {
 		fatalf("Failed to listen: %v", err)
 	}
 
-	s := grpc.NewServer()
+	s := grpc.NewServer(grpc.MaxRecvMsgSize(maxRcvMsgSize))
```

Comment:
Messages are now larger, so I had to increase the max message size (defaults to 4MB). I'm not sure whether that limit would also apply if we were streaming the results.